pkg/vere/io/ames.c

```diff
@@ -2765,6 +2771,60 @@ _ames_recv_cb(uv_udp_t*        wax_u,
   }
 }
 
+static void natpmp_init(uv_timer_t* handle);
+
+static void
+natpmp_cb(uv_poll_t* handle,
+          c3_i        status,
+          c3_i        events)
+{
```

Comment:
We're not checking `status` for errors here. And I'm not sure if `readnatpmpresponseorretry()` can/will handle all cases for us.